src/Pipedrive.net/Clients/OAuthClient.cs

```diff
@@ -83,5 +83,40 @@ public async Task<OAuthToken> RefreshAccessToken(OAuthRefreshTokenRequest reques
             var response = await connection.Post<OAuthToken>(endPoint, body, "application/json", null, hostAddress).ConfigureAwait(false);
             return response.Body;
         }
+        
+        public async Task<OAuthToken> RefreshAccessToken(OAuthRefreshTokenRequest request)
+        {
+            Ensure.ArgumentNotNull(request, nameof(request));
+
+            var endPoint = ApiUrls.OAuthAccessToken();
+
+            var parameters = request.ToParametersDictionary();
+            parameters.TryGetValue("client_id", out var clientId);
+            parameters.TryGetValue("client_secret", out var clientSecret);
+
+            var body = new FormUrlEncodedContent(parameters);
+
+            connection.Credentials = new Credentials(clientId, clientSecret, AuthenticationType.Basic);
+
+            var response = await connection.Post<OAuthToken>(endPoint, body, "application/json", null, hostAddress).ConfigureAwait(false);
+            return response.Body;
+        }
+
+        public async Task RevokeToken(OAuthRevokeTokenRequest request)
```

Comment:
Can you point me to the documentation for revoking the token?

Reply:
Ok, found it https://pipedrive.readme.io/docs/app-uninstallation#section-token-revocation